app/controllers/application_controller.rb

```diff
@@ -8,6 +8,14 @@ class ApplicationController < ActionController::Base
   protected
 
   def configure_permitted_parameters
-    devise_parameter_sanitizer.for(:sign_up) << :username
+    devise_parameter_sanitizer.for(:sign_up) {
+      |u| u.permit(:username, :email, :password, :password_confirmation, :remember_me)
+                                              }
+    devise_parameter_sanitizer.for(:sign_in) {
+      |u| u.permit(:login, :username, :email, :password, :remember_me)
+                                              }
+    devise_parameter_sanitizer.for(:account_update) {
+      |u| u.permit(:username, :email, :password, :password_confirmation, :current_password)
+                                                     }
```

Comment:
`end` at 19, 53 is not aligned with `devise_parameter_sanitizer.for(:account_update) {` at 17, 4.